SecurityPrivateCa/src/V1/Gapic/CertificateAuthorityServiceGapicClient.php

```diff
@@ -18,26 +18,37 @@
 /*
  * GENERATED CODE WARNING
  * Generated by gapic-generator-php from the file
- * https://github.com/google/googleapis/blob/master/google/cloud/security/privateca/v1/service.proto
+ * https://github.com/googleapis/googleapis/blob/master/google/cloud/security/privateca/v1/service.proto
  * Updates to the above are reflected here through a refresh process.
  */
 
 namespace Google\Cloud\Security\PrivateCA\V1\Gapic;
 
 use Google\ApiCore\ApiException;
+use Google\ApiCore\Call;
 use Google\ApiCore\CredentialsWrapper;
-use Google\ApiCore\GapicClientTrait;
 
+use Google\ApiCore\GapicClientTrait;
 use Google\ApiCore\LongRunning\OperationsClient;
-use Google\ApiCore\OperationResponse;
 
+use Google\ApiCore\OperationResponse;
 use Google\ApiCore\PathTemplate;
-use Google\ApiCore\RequestParamsHeaderDescriptor;
 
+use Google\ApiCore\RequestParamsHeaderDescriptor;
 use Google\ApiCore\RetrySettings;
 use Google\ApiCore\Transport\TransportInterface;
 use Google\ApiCore\ValidationException;
 use Google\Auth\FetchAuthTokenInterface;
+use Google\Cloud\Iam\V1\GetIamPolicyRequest;
+use Google\Cloud\Iam\V1\GetPolicyOptions;
+use Google\Cloud\Iam\V1\Policy;
+use Google\Cloud\Iam\V1\SetIamPolicyRequest;
+use Google\Cloud\Iam\V1\TestIamPermissionsRequest;
+use Google\Cloud\Iam\V1\TestIamPermissionsResponse;
+use Google\Cloud\Location\GetLocationRequest;
```

Comment:
These classes need to be generated and likely placed here: https://github.com/googleapis/common-protos-php